config.json

```diff
@@ -467,6 +467,9 @@
             },
             {
               "name": "SetInteriorVehicleData"
+            },
+            {
+              "name": "OnRCStatus"
```

Comment:
@yang1070 fixed with https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/commit/94cbb5adc9c146a65788927aa11bbb889e96b0ab  
since it is just a separate option just to illustrate the flow with ASK_DRIVER access mode and is not directly related to OnRCStatus, I have removed the option.